src/settings/SettingsScreen.js

```diff
@@ -2,7 +2,8 @@
 
 import React, { useCallback } from 'react';
 import type { Node } from 'react';
-
+import { View, Text, Picker } from 'react-native';
```

Comment:
I see a deprecation notice at the top of the [doc](https://reactnative.dev/docs/0.64/picker):

<img width="876" alt="image" src="https://user-images.githubusercontent.com/22248748/164567426-47af0c29-6116-4d28-84e1-82e1c63d473d.png">

> **Deprecated.** Use one of the [community packages](https://reactnative.directory/?search=picker) instead.

From the list at the handy link they give, I think [`@react-native-picker/picker`](https://github.com/react-native-picker/picker) looks like the best choice.